axiom/nr_txn.c

```diff
@@ -3293,6 +3297,90 @@ bool nr_txn_log_decorating_enabled(nrtxn_t* txn) {
   return false;
 }
 
+#define ENSURE_LOG_LEVEL_NAME(level_name) \
```

Comment:
Should a zero length string be converted to "UNKNOWN" as well?  Not clear if this is the same as not having a level defined at all (ie NULL)?

Reply:
I don't know. The spec is not clear about it.

Reply:
Empty log level will be treated as NULL. It's addressed in 3da93909c2cb3520fd2ef16c6340fd58c26a53f5